File: app/src/main/java/com/websarva/wings/android/kaigonote/NyuyokukirokuActivity.java
package com.websarva.wings.android.kaigonote;

import android.os.Bundle;
import android.view.View;

import androidx.appcompat.app.AppCompatActivity;

import com.websarva.wings.android.kaigonote.databinding.NyuyokukirokuBinding;

public class NyuyokukirokuActivity extends AppCompatActivity {
    private NyuyokukirokuBinding binding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = NyuyokukirokuBinding.inflate(getLayoutInflater());
        View view = binding.getRoot();
        setContentView(view);
    }
}

File: app/src/main/java/com/websarva/wings/android/kaigonote/NyuyokukirokuActivity.kt
package com.websarva.wings.android.kaigonote

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import com.websarva.wings.android.kaigonote.databinding.NyuyokukirokuBinding

class NyuyokukirokuActivity : AppCompatActivity() {
    private var binding: NyuyokukirokuBinding? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = NyuyokukirokuBinding.inflate(layoutInflater)
        val view: View = binding!!.root
        setContentView(view)
    }
}